README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
> There are cases where that approach won't work, though: for example, if an upgrade includes a migration that's not backwards compatible. Then the script needs to deploy a new instance without migrations, take the current instance offline (display a maintenance page), run the migrations on the new instance, and point the LB to the new instance, which is a completely different process than we're seeing here. So even if we put steps 3-7 into a command/API, there is a need to document how to do these things via a flexible script.

@bradenmacdonald These also sound like steps that can be automated - and actually, talking about migrations, separating the DB migrations from the VM updates is a good practice that we will need to implement to have a reliable CI workflow. 

Generally, rather than relying on updating a script manually every time, I'd rather look at what needs to be changed from one time to the next in details. I would be surprised if we couldn't find a way to automate everything -- at least on he Ocim part. Ie there could still be work on each release on the git repos Ocim points to, but not to Ocim itself, which needs to be on a more generic level of abstraction, and not require changes on each release of the applications it supports.